integration_tests/models/any_type_column_anomalies.sql

```diff
@@ -1,5 +1,12 @@
+{{ config(
+    materialized="incremental"
+) }}
+
 with training as (
     select * from {{ ref('any_type_column_anomalies_training') }}
+    {% if is_incremental() %}
+        where updated_at > (select max(updated_at) from {{ this }})
```

Comment:
Why not `where 1 = 0` just to clarify?
Also can we add a comment to convey what we're trying to do?